### PR TITLE
Added YCO Pvt. Ltd.

### PR DESCRIPTION
**Pull request by Anupamaraie**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -736,6 +736,22 @@
...
       }
-    }
+    },
+    {
+      "name": "YCo Pvt. Ltd.",
+      "establishedAt": 871582500,
+      "location": "Lokeshwor Tole, Manbhawan, Laltipur, Nepal",
+      "website": "https://yco.com.np/",
+      "email": "info@yco.com.np",
+      "phone": "977 (1) 5551110, 2113000",
```
Comment (thenishantsapkota) [inline data/companies.json:746]: The phone number format is `+977-15551110` and please only add 1 number of the company

File: data/companies.json [hunk 2]
```diff
@@ -736,6 +736,22 @@
         "twitter": "https://twitter.com/yarsalabs/",
         "youtube": "https://www.youtube.com/channel/UCqJAqxSBEmpdCqwgQG3Qoeg"
       }
-    }
+    },
+    {
+      "name": "YCo Pvt. Ltd.",
+      "establishedAt": 871582500,
```
Comment (thenishantsapkota) [inline data/companies.json:742]: What is the source for this?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 1 above)
Comment (Anupamaraie) [reply data/companies.json:746]: okii

File: data/companies.json (hunk 2 above)
Comment (Anupamaraie) [reply data/companies.json:742]: LIS Nepal

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @Anupamaraie!

**Pull request merged**